experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -334,6 +350,99 @@ export class NodeSDK {
     );
   }
 
+  public configureMetricProviderFromEnv(): MetricReader[] {
+    const metricReaders: MetricReader[] = [];
+    const metricsExporterList = process.env.OTEL_METRICS_EXPORTER ?? '';
+    const enabledExporters = filterBlanksAndNulls(
+      metricsExporterList.split(',')
+    );
+
+    // ONLY if OTEL_METRICS_EXPORTER is set
+    if (enabledExporters.length === 0) {
```

Comment:
Hmm, this will always log though even if `OTEL_METRICS_EXPORTER` is undefined. I think it'd probably the best to check if `process.env.OTEL_METRIC_EXPORTER` is `undefined` and returning an empty array right at the beginning of the function to avoid that any log spurious log messages pop up that are not applicable to what the user actually configured.

Reply:
Done